### ISSUE TITLE
radiotray-ng not working on nixos-unstable

### ISSUE THREAD

**Issue by boogewooge** [NixOS/nixpkgs#167864]
### Describe the bug
`radiotray-ng` installs fine, but doesn't work anymore on nixos-unstable. It shows greyed out in the tray. Menus work, but playing a station fails.

Tested on basic gnome and i3 setups.

See the log:
```
[2022-04-08 14:20:46.344050] [0x00007f1b80dc2fc0] [info ] (main.cpp:262) - radiotray-ng (v0.2.8-unknown) starting up
[2022-04-08 14:20:46.344121] [0x00007f1b80dc2fc0] [info ] (main.cpp:264) - radiotray-ng/0.2.8 (Linux x86_64; n/a/n/a (n/a))
[2022-04-08 14:20:46.344128] [0x00007f1b80dc2fc0] [info ] (main.cpp:91) - debug logging enabled
[2022-04-08 14:20:46.344139] [0x00007f1b80dc2fc0] [info ] (player.cpp:26) - starting gstreamer
[2022-04-08 14:20:46.358049] [0x00007f1b80dc2fc0] [error] (player.cpp:427) - could not create souphttpsrc element
[2022-04-08 14:20:46.359375] [0x00007f1b80dc2fc0] [debug] (bookmarks.cpp:31) - loading: /home/boogie/.config/radiotray-ng/bookmarks.json
[2022-04-08 14:20:46.359711] [0x00007f1b80dc2fc0] [debug] (file_monitor.hpp:108) - monitoring: /home/boogie/.config/radiotray-ng/bookmarks.json
[2022-04-08 14:20:46.360763] [0x00007f1b80dc2fc0] [debug] (media_keys.cpp:198) - starting media keys
[2022-04-08 14:21:23.264277] [0x00007f1b80dc2fc0] [info ] (radiotray_ng.cpp:499) - downloading: SomaFM, Groove Salad, https://somafm.com/groovesalad256.pls
[2022-04-08 14:21:23.264331] [0x00007f1b80dc2fc0] [debug] (notification.cpp:63) - notify: Connecting, Radiotray-NG, radiotray-ng-notification
[2022-04-08 14:21:23.273186] [0x00007f1b80dc2fc0] [debug] (playlist_downloader.cpp:140) - http-timeout=15
[2022-04-08 14:21:25.957687] [0x00007f1b80dc2fc0] [debug] (playlist_downloader.cpp:174) - [REDACTED - useless playlist info]
[2022-04-08 14:21:25.958568] [0x00007f1b80dc2fc0] [info ] (playlist_downloader.cpp:96) - decoder: PLS, audio/x-scpls
[2022-04-08 14:21:25.958865] [0x00007f1b80dc2fc0] [error] (player.cpp:88) - gstreamer not ready
[2022-04-08 14:21:25.958873] [0x00007f1b80dc2fc0] [error] (radiotray_ng.cpp:533) - failed to download playlist: https://somafm.com/groovesalad256.pls
[2022-04-08 14:21:25.958994] [0x00007f1b80dc2fc0] [debug] (notification.cpp:63) - notify: Error, Failed to download playlist, radiotray-ng-notification
```

Internet connection is working completely fine and so is the targeted radio stream, so the failed downloads are not due to any external issues.

### Notify maintainers

@dtzWill

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
 - system: `"x86_64-linux"`
 - host os: `Linux 5.17.0-zen1, NixOS, 22.05 (Quokka), 22.05pre367572.b6966d911da`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.7.0`
 - channels(boogie): `"unstable"`
 - channels(root): `"nixos"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by craigem** [comment]
This is true for me too on  NixOS 22.05.

**Comment by a-n-n-a-l-e-e** [comment]
fixed in master and release-23.11 by https://github.com/NixOS/nixpkgs/pull/271975
see also https://github.com/NixOS/nixpkgs/issues/271960
